src/screens/LearningScreen/components/FlashCardContainer.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import { View, Button, Text } from 'react-native';
+
+import { getUserData } from '../../../utils/storage';
+import {
+  getRandomWord,
+  handleCorrectOrIncorrect,
+  getNewLevel,
+} from '../../../utils/spacing';
+import FlashCard from './FlashCard';
+
+const handleRevealButtonPress = (setRevealed) => {
+  setRevealed((flipped) => !flipped);
+};
+
+const handleExpressionsButtonPress = (setExpanded) => {
+  setExpanded((isExpanded) => !isExpanded);
+};
```

Comment:
It looks like these methods aren't used (and they reference undefined values, so I'm guessing they're just leftover code)

Reply:
Looks like they are used, but is there a reason they're defined outside the component?